invenio_communities/templates/semantic-ui/invenio_communities/details/macros/detail.html

```diff
@@ -0,0 +1,15 @@
+{#
+    Copyright (C) 2022 CERN.
+  
+    Invenio RDM Records is free software; you can redistribute it and/or modify
+    it under the terms of the MIT License; see LICENSE file for more details.
+  #}
+
+{% macro truncate_text(text) %}
+  {% if text|length > 200 %}
+    <p id="community-description-less">{{ text|truncate(200)|safe }} <button class="community-description-button">Show more.</button></p>
+    <p id="community-description-all" hidden>{{ text|safe }} <button class="community-description-button">Show less.</button></p>
```

Comment:
the button should be `<a>` tag instead, for reasons mentioned above. Add also accessibility attributes